src/pages/Onboard/AcceptInvitationPage.vue

```diff
@@ -0,0 +1,392 @@
+    slash2Style(i) {
+      let width, top
+      if (i > this.noSlash / 2) {
+        top = i * 5 + 50
+        width = Math.floor(Math.random() * (125 - 75)) + 75
+      } else {
+        top = i * 5
+        width = Math.floor(Math.random() * (75 - 25)) + 25
+      }
+
+      return {
+        top: `${top}%`,
+        height: '120px',
+        left: 0,
+        'transition-delay': `${Math.floor(i / 2) * 25 + 250}ms`,
+        width: `${width}%`
+      }
+    }
+  },
+  apollo: {
+    membershipInvitation: {
+      query: require('@/graphql/Tenant/membership-invitation.gql'),
+      variables() {
+        return { id: this.invitationId }
+      },
+      skip() {
+        return !this.invitationId
+      },
+      loadingKey: 'loadingKey',
+      pollInterval: 5000,
+      update: data => data.membership_invitation_by_pk
+    }
+  }
+}
+</script>
+
+<template>
+  <v-container
+    class="ma-0 pa-0 position-relative h-100 bg-grey overflow-y-hidden"
+    fluid
+  >
+    <transition-group name="fade" mode="out-in">
+      <div
+        v-if="$vuetify.breakpoint.smAndUp"
+        key="slash-1"
+        class="slash-blue o-slash slash-1"
+        :class="[slash1Class, $vuetify.breakpoint.smAndUp ? 'slash-1' : '']"
+      >
+      </div>
+      <div
+        key="slash-3"
+        class="slash-grey o-slash slash-3"
+        :class="[slash3Class, $vuetify.breakpoint.smAndUp ? 'slash-3' : '']"
+      >
+      </div>
+      <div
+        v-if="$vuetify.breakpoint.smAndUp"
+        key="slash-4"
+        class="slash-grey o-slash slash-4"
+        :class="slash4Class"
+      >
+      </div>
+      <div
+        v-if="$vuetify.breakpoint.smAndUp"
+        key="slash-5"
+        class="slash-orange o-slash slash-5"
+        :class="slash5Class"
+      >
+      </div>
+
+      <div v-if="$vuetify.breakpoint.smAndUp" key="name-team-slashes">
+        <div
+          v-for="i in noSlash"
+          :key="`name-team-slash-${i}`"
+          class="o-slash slash-2"
+          :class="[
+            slash2Class,
+            i % 2 === 0 ? 'slash-grey' : 'slash-grey-reverse'
+          ]"
+          :style="slash2Style(i)"
+        ></div>
+      </div>
+    </transition-group>
+    <v-container
+      v-if="!loadingPage"
+      class="position-absolute onboard-content text-center pa-0"
+      fluid
+    >
+      <v-row v-if="invitationError" align="center">
+        <v-col class="grey--text text--lighten-5 mx-12">
+          <div class="display-1">
+            <span> {{ mutationErrorMessage || errorMessage }} </span>
+          </div>
+          <div>
+            <v-btn class="mt-8" color="primary" x-large @click="toDashboard">
+              Back to the dashboard!
+              <v-icon right>fas fa-rocket</v-icon>
+            </v-btn>
+          </div>
+        </v-col>
+      </v-row>
+      <v-row v-else align="center">
+        <v-col>
+          <div class="display-1 grey--text text--lighten-5">
+            You've been invited to join {{ teamName }}!
```

Comment:
Let's make the team name stand out more, maybe add a semi-bold class to it and/or color the text?